requirements-dev.txt

```
@@ -0,0 +1,6 @@
+pytest==7.1.0
+pytest-cov==3.0.0
+pytest-mock==3.7.0
+pytest-randomly==3.11.0
+typing-extensions==4.1.1
+black==22.1.0
```

It would be nice to keep this listed sorted.
```suggestion
black==22.1.0
pytest==7.1.0
pytest-cov==3.0.0
pytest-mock==3.7.0
pytest-randomly==3.11.0
typing-extensions==4.1.1
```